### PR TITLE
Add instruction for "Application Default Credentials" to run e2e tests locally

### PR DESCRIPTION
**Pull request by YangLu1031**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
We missed the instruction to setup "Application Default Credentials" to run e2e tests locally.

**Does this PR introduce a user-facing change?**:

```release-note
Add instruction to setup "Application Default Credentials" to run GCE Windows e2e tests locally. 
```

### CODE REVIEW

File: cluster/gce/windows/README-GCE-Windows-kube-up.md [hunk 1]
```diff
@@ -23,6 +23,13 @@ rm -rf ./kubernetes/; rm -f kubernetes.tar.gz; rm -f ~/.kube/config
 # if you're working with multiple projects and don't want to repeatedly switch
 # between gcloud config configurations.
 export CLOUDSDK_CORE_PROJECT=<your_project_name>
+
+# To run e2e test locally, make sure "Application Default Credentails" is set in any of the places:
```
Comment (yujuhong) [inline cluster/gce/windows/README-GCE-Windows-kube-up.md]: nit: add a link to the official documentation in case this changes.

Comment (pjh) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: https://cloud.google.com/sdk/docs/authorizing#authorizing_with_a_service_account is the right link for this

**Review by pjh — APPROVED** [review]

File: cluster/gce/windows/README-GCE-Windows-kube-up.md (hunk 1 above)
Comment (YangLu1031) [reply cluster/gce/windows/README-GCE-Windows-kube-up.md]: Thanks @pjh, found this link, is it better? http://cloud/sdk/gcloud/reference/auth/application-default/

**Comment by yujuhong** [comment]
The failure is legit: 
```
cluster/gce/windows/README-GCE-Windows-kube-up.md:27:58: "Credentails" is a misspelling of "Credentials"
```

**Comment by yujuhong** [comment]
/lgtm
/approve
Thanks!

**Pull request merged**